README.md

```diff
@@ -820,11 +820,14 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 6.12. Limit the allowed login requests of each user
+## ![✔] 6.12. Prevent brute-force attacks against authorization
```

Comment:
Maybe we could add "endpoints" to this headline, so "Prevent brute-force attacks again authorization endpoints", what do you think?

Reply:
Let's keep it as short as possible. This phrase has the same meaning with or without "endpoints" word in the end.

Reply:
I don't think it's complete in that case though. For the sake of readability I would vouch for adding endpoints, one additional word doesn't change the whole thing. But I'd like to get more opinions on this, what do you think @sagirk @i0natan @js-kyle ?

Reply:
@BrunoScheufler Not all frameworks/libraries use "endpoint" term. For example, developers name it **authorization resolver** in GraphQL. It may be just **authorization service** in service oriented architecture. If we do not add it, title would be wider.